tests/wpt/web-platform-tests/webaudio/the-audio-api/the-audiobuffersourcenode-interface/audiobuffersource-multi-channels.html

```diff
@@ -33,8 +33,8 @@
               context.decodeAudioData(arrayBuffer).then(audioBuffer => {
                 expectedAudio = audioBuffer;
                 task.done();
-              }).catch(() => {
-                assert_unreached("arrayBuffer should be triggered correctly");
+              }).catch(error => {
```

Comment:
I squashed the fix into wrong commit 😭 Will update to make it be in the right commit.